service/e2e_tests/test_search.py

```diff
@@ -207,39 +220,39 @@ def test_filter_search(self):
         # Locally the test passes.
         # The tests fail with a TimeoutException without a message so it's not entirely clear what goes wrong here.
         # We should re-enable this test.
-        self.skipTest("Skipped due to strange Github behaviour")
```

Comment:
Dit moet je ook niet committen. Heb dit ooit gecommit en toen gingen de tests weer vaker mis. Toen heb ik het gerevert. Niet helemaal duidelijk waarom jij het nu weer opneemt. Ik vermoed per ongeluk, dus lijkt mij dat we dit niet moeten mergen. Laat het weten als je er anders over denkt.